packages/nimble-components/src/button/specs/button-visual-states.md

```diff
@@ -0,0 +1,45 @@
+# Nimble-Button New Visual Designs Implementation
+
+## Problem Statement
+
+The Visual Design spec for Nimble buttons now includes primary button states `primary` and `primary-accent` that aren't implemented in the nimble-button component yet.
+
+## Links To Relevant Work Items and Reference Material
+
+[Nimble Issue #663: Adopt nimble-button visual designs for states primary, quiet, and loud](https://github.com/ni/nimble/issues/663)
+
+[Figma Spec](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-77205)
+
+## Implementation / Design
+
+There are three possible values for `appearance-variant` on a button: `default`, `primary`, and `primary-accent`, each with its own color scheme based on the UI theme.
+`primary` and `primary-accent` states of buttons are primary buttons, which can be used together up to 3 times on a page.
+
+The `primary` button can be used when there is a conflict with color and its context.
+The `primary-accent` button can be used in situations where a button needs to have the most prominent eye-catching approach, or when there is a lack of color.
+
+Ghost buttons will not have `primary` or `primary-accent` states.
+Outline and Block buttons in the Color UI will not have a `primary-accent` state, as this has been declared as not recommended for this type of background.
+
+These limitations will be included in the description of `appearance-variant`, and if their values are selected, the button will change to the `default` value for `appearance-variant`.
+
+The current `appearance-variant` attribute will be used to hold these new states, as its previous implementation purpose is no longer used.
```

Comment:
It would be good to add a sentence here about continuing to use the `appearance-variant` attribute with a valid value of `primary` means that this will not be a backwards-compatibility breaking change.

Maybe something like:
_Existing instances of `primary` buttons will automatically use the new `primary` button styling with no modifications required._